NewEmuChecklist.txt

```diff
@@ -0,0 +1,75 @@
+RetroAchievements Emulator Requirements:
+* If the emulator does not support a feature, the requirement is automatically met. This is primarily a set of
+  guidelines to ensure the interaction between the emulator and the toolkit is consistent between emulators.
+* If the system is supported by RetroArch, explain why a standalone emulator is desirable. RetroArch support is 
+  preferred as it is usable by more players (i.e. not just Windows users), and adding support to RALibRetro for 
+  an additional core is much easier than overhauling an entire emulator.
+
+[ ] Game Identification
+  [ ] Hash method documented and approved
+  [ ] If emulator supports dynamic patching [i.e. ips], hash is calculated after applying patch
+  [ ] Hash ignores any persistant save data [SRAM]
+  [ ] Game can be properly identified (RA_OnLoadNewRom)
+  [ ] Unidentified game provides some descriptor [typically filename] to unknown game dialog (RA_InstallSharedFunctions>RA_GetEstimatedGameTitle)
+  [ ] Unidentified game provides list of known games for console (RA_SetConsoleID)
+[ ] Emulator
+  [ ] DLL is downloaded if not available (RA_Init - new enum may be needed)
+  [ ] RetroAchievements menu is visible (RA_CreatePopupMenu) and functional (RA_InvokeDialog)
+  [ ] RetroAchievements menu updates when logging in/out (RA_InstallSharedFunctions>RA_RebuildMenu)
+  [ ] Login dialog displayed on first run or auto-login on future runs (RA_AttemptLogin)
+  [ ] Title includes logged in user (RA_UpdateAppTitle)
+  [ ] RALog.txt ends with "Shutdown complete" after closing emulator (RA_Shutdown)
```

Comment:
Just nitpicking, but I think behavior (e.g. RA resources are disposed on exit) would act as better criteria than logging output.

Reply:
disposing the resources is definitely the intended reason for calling `RA_Shutdown`, but the log message is the easiest way to verify that it's happening.